awx/settings/defaults.py

```diff
@@ -198,7 +198,7 @@ def IS_TESTING(argv=None):
 
 # The number of seconds (must be an integer) to buffer callback receiver bulk
```

Comment:
Please update this if you're changing it as below

Reply:
> must be an integer

...and now I remember why I changed this.  The `timeout=` argument to the redis `blpop` must be an integer.

https://github.com/ansible/awx/pull/9086/files#diff-915de8c0ee78894addb6ce93fd72bda6adc7b9a0bf2ba54ae9e9f1a6c0d8f011R56

This PR probably still needs some adjusting and testing on my end, but I've got an idea on what to adjust now.